frame/identity/src/benchmarking.rs

```diff
@@ -200,13 +201,13 @@ benchmarks! {
 			let caller: T::AccountId = whitelisted_caller();
 			let _ = add_sub_accounts::<T>(&caller, s)?;
 		};
-		let x in 1 .. T::MaxAdditionalFields::get() => {
-			// Create their main identity with x additional fields
-			let info = create_identity_info::<T>(x);
-			let caller: T::AccountId = whitelisted_caller();
-			let caller_origin = <T as frame_system::Config>::Origin::from(RawOrigin::Signed(caller));
-			Identity::<T>::set_identity(caller_origin, Box::new(info))?;
-		};
+		let x in 1 .. T::MaxAdditionalFields::get();
+
+		// Create their main identity with x additional fields
```

Comment:
this is logically different.

The `let x in 1..10 { code}` i believe runs a loop over the wrapped code.

Reply:
Reading [this](https://github.com/paritytech/substrate/blob/9baf5da98eb3c7c5c35398b4d52c54a53fe52bf2/frame/benchmarking/src/lib.rs#L68-L86) and the real macro code, I don't think this is a loop. It just declares the parameter room and then we benchmark with the combination of all of these parameters?